test/logging.test.js

```diff
@@ -26,12 +26,12 @@ const logEmitterEmitSpy = jest.spyOn(logEmitter, 'emit')
 
 const exampleErrorLog = [
```

Comment:
Did tests using this object pass previously? Doesn't look like it should as we are patching the test input.

Reply:
So these tests were the issue - both [contentful-export](https://github.com/contentful/contentful-export/blob/6bfb1968d6fab64a7203a5220d60e7df8d3ddd09/lib/index.js#L172) and contentful-import rely on the assumption that `ts` is a string. I believe the tests are wrong in this instance as they were creating this test data using type `number`.